code/ensemble/manual_exclusions.csv

```diff
@@ -0,0 +1,2 @@
+﻿forecast_date,model
```

Comment:
I've had a look through the tagged ensemble commits. I think we also want 
`2021-03-08,ITWW-county_repro,"late submission"`